src/login/tests/LoginPage.test.jsx

```diff
@@ -230,9 +279,9 @@ describe('LoginPage', () => {
       },
     });
 
-    const loginPage = mount(reduxWrapper(<IntlLoginPage {...props} />));
-    expect(loginPage.text().includes('Company or school credentials')).toBe(false);
-    expect(loginPage.text().includes('Or sign in with:')).toBe(false);
+    const { queryByText } = render(reduxWrapper(<IntlLoginPage {...props} />));
+    expect(queryByText('Company or school credentials')).toBeNull();
```

Comment:
Can we use screen functions for the element selector to follow best practice?

Reply:
I have used it but it did not workout that is why I had to use it